src/hpc-cache/azext_hpc_cache/_params.py

```diff
@@ -31,6 +31,11 @@
 
 storage_target_type = CLIArgumentType(help='Name of the Storage Target.')
 
+nfs3_usage_model_type = CLIArgumentType(
+    help='Identify the primary usage model to be used for this Storage Target. '
+         'You can use command `az hpc-cache usage-model list` to see the names of usage models. '
+         'For more details about usage models, please refer to https://docs.microsoft.com/en-us/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model')

```

Comment:
Please remove the locale code 'en-us' from the URL - change to:
https://docs.microsoft.com/azure/hpc-cache/hpc-cache-add-storage?tabs=azure-cli#choose-a-usage-model

Reply:
OK~ Thanks for reminding